llvm.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# This script will install the llvm toolchain on the different 
```

Comment:
What about adding a license? (probably the same as llvm)

Reply:
copied the license statement over from the LLVM source code